packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -46,6 +46,10 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
         />
     );
 
+    const modalStateHandler = value => {
+        my_profile_store.setModalOpenState(value);
+    };
```

Comment:
can we use `should_show_popup` to check if modal is open instead of creating another function?

Reply:
We can use. Here I am using the the state of modal as a check to open the subsequent modal